src/components/error/homeButton.tsx

```diff
@@ -0,0 +1,15 @@
+import Link from "next/link";
+const HomeButton = () => {
+  return (
+    <div className="flex items-center justify-center">
+      <Link
+        href="/"
+        className="mt-[57vh] rounded-full border bg-gradient-to-r from-red-300 to-purple-300 px-6 py-2 text-3xl"
```

Comment:
```suggestion
        className="rounded-full border bg-gradient-to-r from-red-300 to-purple-300 px-6 py-2 text-3xl"
```
we don't need this, we will just center everything in the page. Rename your file to NotFound.tsx as well. your text is also the wrong color, change it to white. The text is also incorrect and a shadow is missing, feel free to reference tailwind documentation for that. The colors used are also wrong, please reference the tailwind config colors for cvdsa